src/ansiblelint/rules/yaml_rule.py

```diff
@@ -177,5 +175,5 @@ def test_yamllint_has_help(default_rules_collection: RulesCollection) -> None:
                 assert collection.help is not None
```

Comment:
Can you also please rename `collection` variable to `rule`, I don't know how this code went in because we are talking about a rule. The bug is unrelated to your change but is in the same area, so better to address it.

Reply:
ack